meson.build

```diff
@@ -1566,6 +1568,11 @@ if features['macos-cocoa-cb']
     swift_sources += files('video/out/cocoa_cb_common.swift',
                            'video/out/mac/gl_layer.swift')
 endif
+if features['cocoa'] and features['vulkan'] and swift.allowed()
```

Comment:
Maybe replace all usage of `swift.allowed()` with `features['swift']`? I know it's not really related to this PR specifically.

Reply:
done, was wondering what the preferred way is. also changed one of the cocoa checks.